src/rgw/rgw_sal_motr.cc

```diff
@@ -1546,7 +1546,11 @@ int MotrStore::initialize(CephContext *cct, const DoutPrefixProvider *dpp) {
 int MotrStore::create_gc() {
   int ret = 0;
   motr_gc = std::make_unique<MotrGC>(cctx, this);
-  motr_gc->initialize();
+  ret = motr_gc->initialize();
+  if (ret < 0) {
+    // Failed to initialize MotrGC
```

Comment:
Add here log statement (with 0 level)

Reply:
No need to log here. Logging is already done at the caller side. Refer int MotrStore::initialize()